advocacy_docs/tools/alteruser_utility/index.mdx

```diff
@@ -0,0 +1,213 @@
+ 
+ Assigns the LOGIN privilege to an existing role, so it can log in.
+
+`-L` or `--no-login`            
+
+ Removes the LOGIN privilege from an existing role, so it can't login.
+  
+`-P` or `--pwprompt`            
+
+ Assigns a password to an existing role.
+  
+`-r` or `--createrole`          
+
+ Assigns the CREATEROLE privilege to an existing role, so it can create, alter, drop, comment on, and change the security label for other roles.
+  
+`-R` or `--no-createrole`       
+
+ Removes the CREATEROLE privilege from an existing role, so it can't create new roles.
+  
+`-s` or `--superuser`           
+
+ Assigns the SUPERUSER privilege to an existing role.
+  
+`-S` or `--no-superuser`        
+
+ Removes the SUPERUSER privileges from an existing role.
+  
+`-v` or `--valid-until=TIMESTAMP`
+                            
+ Sets a date and time after which the role's password is no longer valid.
+  
+`-V` or `--version`             
+
+ Prints the `alteruser` version information and exits.
+  
+`--interactive`             
+
+ Prompts for the missing role name and attributes if none are specified on the command line.
+
+`--bypassrls`               
+
+ Allows an existing role to bypass row-level security (RLS) policy.
+  
+`--no-bypassrls`            
+ 
+ Disallows an existing role to bypass row-level security (RLS) policy.
+  
+`--replication`             
+
+ Assigns REPLICATION privilege to an existing role, so it can initiate the replication.
+  
+`--no-replication`          
+ 
+ Removes REPLICATION privilege from an existing role, so it can't initiate replication.
+  
+`--account=MODE`            
+
+ Allows an existing role to change the account mode. The mode can be LOCK or UNLOCK.
+  
+`--createdblink`            
+
+ Assigns the CREATE DATABASE LINK privilege to an existing role, so it can create database links.
+  
+`--no-createdblink`         
+
+ Removes the CREATE DATABASE LINK privilege from an existing role, so it can't create database links.
+
+`--createpublicdblink`      
+
+ Assigns the CREATE PUBLIC DATABASE LINK privilege to an existing role, so it can create public database links.
+
+`--no-createpublicdblink`   
+
+ Removes the CREATE PUBLIC DATABASE LINK privilege from an existing role, so it can't create public database links.
+  
+`--droppublicdblink`        
+
+ Assigns the DROP PUBLIC DATABASE LINK privilege to an existing role, so it can drop public database links.
+
+`--no-droppublicdblink`     
+
+ Removes the DROP PUBLIC DATABASE LINK privilege from an existing role, so it can't drop public database link.
+  
+`--encrypt-password=METHOD`
+
+ Assigns an encryption method for a password. Use this option with the `--pwprompt` option. The encryption method can be "scram-sha-256", "md5", "server_encryption", or "plain" (default). If you select "server_encryption", the password encrypts using the server's `password_encryption` GUC value. If you select "plain", the password isn't encrypted and is sent in plain text.
```

Comment:
Are these double quotes correct? If so, should they also be tagged?

Reply:
tagged them